### DISCUSSION TITLE
copilot.vim: Repeating "access ending" alert

### DISCUSSION THREAD

**Discussion by knowald** [community/community#30378] [Copilot Conversations]
Neovim with the Copilot plugin is currently interrupting me every time a newly launched nvim enters the Insert mode:

```
Thank you for using GitHub Copilot. Your extended access ends in one day.

Request Actions:
1. Copilot Settings
2. Dismiss
Type number and <Enter> or click with the mouse (q or empty cancels):
```

I am currently on the monthly plan. It would also be nice to remember the first "Dismiss" anyway 👍

**Comment by hrkrx** [comment]
Same Issue:

https://github.com/orgs/community/discussions/30370
